UnityProject/Assets/Scripts/Chat/Chat.Process.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using UnityEngine;
 using System.Text.RegularExpressions;
 using System.Globalization;
+using System.Linq;
```

Comment:
Hope adding Linq is OK, haven't seen it discussed much. Felt like it was (in this case) easier to code & understand than regex .